test/egg.test.js

```diff
@@ -415,6 +415,10 @@ describe('test/egg.test.js', () => {
       const wrapped = app.toAsyncFunction(fn);
       return wrapped(true).then(res => assert(res === true));
     });
+
+    it('not translate an obj that is not a function', () => {
+      assert.throws(() => app.toAsyncFunction(true));
```

Comment:
should assert the error message.

Reply:
Well……it seems ambigious in Nodejs10? I just wrote the err msg but failed to pass in Nodejs 10?

Reply:
I'll have a try again :)